### ISSUE TITLE
Newline added to password when attempting git-svn checkout over HTTP(S)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#1412]
```
What steps will reproduce the problem?
1. Right-click explorer window, select "Git Clone..."
2. Enter URL for Subversion repo running across HTTP/HTTPS
3. Check SVN-related boxes, including "Username"
4. Enter a valid username
5. Click "OK"
6. When the password prompt appears, check the username (it's correct) and 
enter a valid password
7. Auth fails and you get asked two more times until it fails for good.

What is the expected output? What do you see instead?
After step 6, I expect it to authenticate successfully

What version of TortoiseGit and msysgit are you using? On
what operating system?
TortoiseGit 1.7.12.9 
git version 1.7.11.msysgit.1
Windows 7 64-bit

Please provide any additional information below.
When watching the auth process via Wireshark and using a test PHP script on the 
server side, it is clear that some part of the auth process is adding a newline 
to the end of the password. As such, the password fails to pass (since it's 
been modified) and the whole process fails. I poked around in some of the Perl 
scripts (like git-svn) but was unable to find the source of this error.

I tried adding the username:password prefix to the HTTP(S) request, but 
apparently git-svn ignores that. It kept trying to use my Windows username, 
which isn't the one I want to use.

tl/dr: Let's say I enter my password as "password123". TortoiseGIT sends 
"password123\n", which is wrong.
```

Original issue reported on code.google.com by `courtarro` on 26 Sep 2012 at 7:04
- Merged into: #967
